applications/GeoMechanicsApplication/custom_processes/apply_component_table_process.h

```diff
@@ -0,0 +1,55 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Ignasi de Pouplana,
+//                   Vahid Galavi
+//
+
+#pragma once
+
+#include "includes/kratos_export_api.h"
+#include "includes/kratos_parameters.h"
+#include "includes/model_part.h"
+#include "includes/smart_pointers.h"
+#include "includes/table.h"
+#include "processes/process.h"
+
+#include <string>
+
+namespace Kratos
+{
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) ApplyComponentTableProcess : public Process
+{
+public:
+    KRATOS_CLASS_POINTER_DEFINITION(ApplyComponentTableProcess);
+
+    using TableType = Table<double, double>;
+
+    ApplyComponentTableProcess(ModelPart& model_part, Parameters rParameters);
```

Comment:
Let's make the parameter names comply with the Kratos Style Guide:
```suggestion
    ApplyComponentTableProcess(ModelPart& rModelPart, Parameters ProcessSettings);
```

Reply:
Done